src/electionguard-ui/ElectionGuard.UI/Views/LoginPage.xaml

```diff
@@ -15,15 +15,11 @@
         <Entry
             windows:VisualElement.AccessKey="N"
             Placeholder="{helper:Translate LoginPlaceholder}"
+            ReturnCommand="{Binding LoginCommand}"

```

Comment:
handles typing name and hitting enter on keyboard

Reply:
🙌🏼